zebra/rt_netlink.c

```diff
@@ -755,9 +755,36 @@ static int netlink_route_change_read_unicast(struct nlmsghdr *h, ns_id_t ns_id,
 				nh.bh_type = bh_type;
 			}
 			nh.ifindex = index;
+
+			if (index) {
+				/*
+				 * Yes we are looking this up
+				 * for every nexthop and just
```

Comment:
didn't quite get this comment - aren't you only doing this in the NOT-multipath case?

Reply:
I cut-n-pasted the same bit of code from above which produced this fun.

Reply:
if the comment is not correct, even if it came from somewhere else, could it be removed or ... corrected?